week1/assignments/01_html_to_html5.html

```diff
@@ -19,23 +22,28 @@
           - may be want to set default text color to #333
           - set the gap between two lines by setting line-height to 1.7
         */
+        font-family: 'Times New Roman', Times, serif;
```

Comment:
This need to be different font family like `Helvatica, Arial, San serif`. As font are `Times New Roman` by default.